tests/run-pass/ref-invalid-ptr.rs

```diff
@@ -1,12 +1,12 @@
-// FIXME validation disabled because it checks these references too eagerly
+// FIXME: validation disabled because it checks these references too eagerly.
 // compile-flags: -Zmiri-disable-validation
 
 fn main() {
     let x = 2usize as *const u32;
-    // this is not aligned, but we immediately cast it to a raw ptr so that must be okay
+    // This is not aligned, but we immediately cast it to a raw ptr so that must be ok.
```

Comment:
This strikes me as odd, why use the abbreviation?

Reply:
It is a bit, yes, but this is just to bring it in line with the Rustc codebase. (Personally, I avoid the word in code.)